db/migrate/20151116105412_add_triggering_to_job_invocation.rb

```diff
@@ -0,0 +1,9 @@
+class AddTriggeringToJobInvocation < ActiveRecord::Migration
+  def up
+    add_column :job_invocations, :triggering_id, :integer, :index => true
```

Comment:
Again, yes. A job invocation could be created without a triggering so it made sense to me not to enforce the association